packages/config/config/devices/0x027a/zse29.json

```diff
@@ -16,6 +16,96 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "PIR Control",
+			"maxNodes": 4
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Value",
+			"description": "Set the value of Basic Set commands sent to the devices associated in Group 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Enable / Disable Motion Reports",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "Motion Sensitivity",
+			"description": "Adjust motion sensitivity where 10 is the most sensitive setting.",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Lux Level Threshold",
+			"description": "Set lux level threshold with 10 being the lowest reported value.  The sensor will report motion only if lux level is below the set value. 0=set manually; 1=ignore lux.",
+			"unit": "lux",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 900,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "Motion Re-trigger Time",
+			"description": "Delay before the sensor reports no motion after detecting the last motion activity. 0=set manually.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 720,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "Lux Reporting Frequency",
+			"unit": "minutes",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 1440,
+			"defaultValue": 30,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		}
```

Comment:
You're missing parameters. Parameter 7 is a read-only report of the motion knob. Parameter 8 is a read-only report of the physical knob setting. https://www.support.getzooz.com/kb/article/321-zse29-outdoor-motion-sensor-ver-2-0-advanced-settings/

You need to make some of these parameters as conditional on the FW version, using the new conditional configuration setting. https://www.support.getzooz.com/kb/article/307-zse29%C2%A0outdoor-motion-sensor-change-log/

Conditional values are described here. Don't worry about templating, I'll handle that. https://zwave-js.github.io/node-zwave-js/#/development/config-files

Reply:
Added the read-only parameters.  Will have to work on conditional FW templating later.  (Can I place the individual conditional parameters right in the middle of the parameter specifications, or do I need to completely reproduce the whole parameter set for each firmware version?)

Reply:
You can just add conditions for individual parameters. If a parameter changes between versions, you need to use an array of objects for that one parameter instead.

Reply:
I limited the parameters to FW >= 2.0.  I started to include conditional parameters for FW 1.3, but it's unclear exactly how they're implemented prior to 2.0 (the parameters are described in the documentation, but I'm not sure whether the numbering was the same).  Parameters prior to 2.0 were quite limited in any event.